src/components/PurchaseOrder/PO.js

```diff
@@ -405,6 +423,20 @@ class PO extends Component {
           </IfPermission>
 
           <IfPermission perm="orders.item.put">
+            <IfPermission perm="orders.item.approve">
+              {isApprovalRequired && !isApproved && (
```

Comment:
`isApprovalRequired && !isApproved` looks confusing, could you please leverage this to one boolean?